power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -665,15 +665,12 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
 
     template <symmetry_tag sym>
     auto calculate_power_flow(double err_tol, Idx max_iter, CalculationMethod calculation_method) {
-        auto const result_pf =
+        auto result_pf =
```

Comment:
this `const` prevents moving and therefore causes a full copy of the entire powerflow output